docs/dev/development-setup.md

```diff
@@ -0,0 +1,91 @@
+# Development setup
+
+## Forking
+
+Please note that Go requires code to live under absolute paths, which complicates forking. While a fork lives at `https://github.com/<github-handle>/ibc-go`, the code should never exist at `$GOPATH/src/github.com/<github-handle>/ibc-go`. Instead, we use `git remote` to add the fork as a new remote for the original repo `$GOPATH/src/github.com/cosmos/ibc-go`), and do all the work there.
+
+For instance, to create a fork and work on a branch of it, we would:
+
+- Create the fork on GitHub, using the fork button.
+- Go to the original repo checked out locally (i.e. `$GOPATH/src/github.com/cosmos/ibc-go`)
+- `git remote add fork git@github.com:<github-handle>/ibc-go.git`
+
+Now `fork` refers to the fork and `origin` refers to the ibc-go version. So we can `git push -u fork main` to update the fork, and make pull requests to ibc-go from there. Of course, replace `<github-handle>` with your git handle.
+
+To pull in updates from the origin repo, run
+
+- `git fetch origin`.
+- `git rebase origin/main` (or whatever branch you want).
+
+## Dependencies
+
+We use [Go 1.14 Modules](https://github.com/golang/go/wiki/Modules) to manage dependency versions.
+
+The main branch of every Cosmos repository should just build with `go get`, which means they should be kept up-to-date with their dependencies, so we can get away with telling  people they can just `go get` our software.
```

Comment:
```suggestion
The main branch of every Cosmos repository should just build with `go get`, which means they should be kept up-to-date with their dependencies, so we can get away with telling people they can just `go get` our software.
```

Reply:
I'm not sure how useful this section is. I feel like `go get` is something every go module makes use of?